lms/static/sass/views/_program-marketing-page.scss

```diff
@@ -1,1767 +1,179 @@
-                display: block;
-                font-size: 0;
-                margin-top: 5px;
-
-                > span {
-                    display: inline-block;
-                    text-decoration: none;
-                    font-size: 1rem !important;
-                    &:first-child {
-                        border-right: 1px solid #4a4a4a;
-                        padding-right: 6px;
-                        margin-right: 6px;
-                        @media (min-width: 768px) {
-                            padding-right: 10px;
-                            margin-right: 10px;
-                        }
-                    }
-                }
-            }
-        }
-
-        p {
-            margin-bottom: 0;
-            font-weight: normal;
-            font-size: 1rem;
-        }
-
-        &:last-child {
-            margin-bottom: 0;
-        }
-    }
-
-    .courses-in-program.container {
-        display: block;
-        padding: 20px 0;
-        background: #ececec;
-        margin-top: 15px;
-        @media (min-width: 768px) {
-            margin-top: 0;
-            background: $white;
-            padding: 20px;
-        }
+#program-details-page{
+  font-size: 16px;
 
-        .course-cards {
-            padding-top: 5px;
-            @media (min-width: 768px) {
-              width: 100% !important;
-              transform: none !important;
-            }
-        }
-
-        > .hd {
-            padding: 0 1.25rem !important;
-            margin-bottom: 15px;
-            @media (min-width: 768px) {
-              padding: 0;
-              font-size: 18px !important;
-            }
-            @media (min-width: 1024px) {
-                font-size: 22px !important;
-            }
-        }
-
-        .course-card {
-            margin-bottom: 20px;
-            min-height: 283px;
-            position: relative;
-            @media (max-width: 767px) {
-                min-height: 300px;
-                margin-left: 5px;
-                margin-right: 5px;
-                padding-bottom: 50px;
-                overflow: hidden;
-                position: relative;
-            }
-
-            .tbl-view {
-                .tbl-col {
-                    .btn-enroll-now {
-                        position: absolute;
-                        left: 0;
-                        right: 0;
-                        bottom: 0;
-                        width: 100%;
-                        border-radius: 0;
-                        line-height: 22px;
-                        background: #0f8012;
-                        @media (min-width: 768px) {
-                            position: relative;
-                            border: 1px solid #ccc;
-                            max-width: 200px;
-                        }
-                    }
-
-                    &:first-child {
-                        @media (min-width: 768px) {
-                            width: 75%;
-                        }
+  h1{
```

Comment:
Don't target elements directly as this will affect all such elements on the page. Our best practice is to only target classes.